sdk/core/azure-core-amqp/samples/internal/eventhub_async_writer_sample/eventhub_async_writer_sample.cpp

```diff
@@ -17,19 +20,27 @@ int main()
           EH_CONNECTION_STRING)};
   std::string targetUrl
       = "amqps://" + credentials->GetHostName() + "/" + credentials->GetEntityPath();
+  std::string targetEntity = credentials->GetEntityPath();
+  if (targetEntity.empty())
+  {
+    targetEntity = GetEnvHelper::GetEnv("EVENTHUB_NAME");
```

Comment:
That PR exists - see https://github.com/Azure/azure-sdk-for-cpp/pull/4655

I've not removed existing uses because it introduces a behavior change (so I'd like to have SDK owners make that change), but I've removed `get_env.hpp` from all the samples that don't actually use `GetEnvHelper` (in other words, from the samples that included `get_env.hpp` as a replacement for `#define _CRT_SECURE_NO_WARNINGS`).

I also cleaned up `get_env.hpp` so it doesn't include so many things, which will allow us to remove the get_env.hpp include file rule.